### PR TITLE
Modify PR preview directory

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

---

**Related issue :**

### CODE REVIEW

**Review by gwarf — COMMENTED** [review]
So you mean that the zip removes the parent `public` directory that should contain the built code + the PR number?
If that's the case you will have to also adapt the part that is reading the PR number when adding the comment.

**Comment by gwarf** [comment]
I'm a bit unsure, see 
Screenshot 2021-08-24 at 12 33 55

**Comment by gwarf** [comment]
From #297: https://github.com/EGI-Federation/documentation/runs/3410041245?check_suite_focus=true#step:3:4

**Comment by enolfc** [comment]
@gwarf can you take another look?

File: .github/workflows/deploy_pr_preview.yml [hunk 1]
```diff
@@ -44,7 +44,11 @@ jobs:
             fs.writeFileSync('${{github.workspace}}/pr.zip', Buffer.from(download.data));
 
       - name: Extract pull request preview
-        run: unzip pr.zip
+        id: extract
+        run: |
+          unzip pr.zip
+          PR_NUMBER="$(cat ./PR")
```
Comment (gwarf) [inline .github/workflows/deploy_pr_preview.yml:50]:
```suggestion
          PR_NUMBER="$(cat ./PR)"
```

**Review by gwarf — APPROVED** [review]
Let's try this.

**Pull request merged**